R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1804,11 +1804,11 @@ test_that("describe() and summarize() on a DataFrame", {
   expect_equal(collect(stats)[2, "age"], "24.5")
   expect_equal(collect(stats)[3, "age"], "7.7781745930520225")
   stats <- describe(df)
-  expect_equal(collect(stats)[4, "name"], "Andy")
+  expect_equal(columns(stats), c("summary", "age"))
```

Comment:
Sorry one last thing - instead of removing the previous test can we add a new assert ? 
Also maybe can we add the failing test case from the JIRA as a new test case ?